src/explore-education-statistics-common/src/modules/charts/util/getMinorAxisSize.ts

```diff
@@ -0,0 +1,38 @@
+import { DataSetCategory } from '@common/modules/charts/types/dataSet';
+import parseNumber from '@common/utils/number/parseNumber';
+
+interface Props {
+  dataSetCategories: DataSetCategory[];
+  minorAxisSize: number | undefined;
+  minorAxisDecimals: number | undefined;
+  minorAxisUnit: string;
+}
+
+/**
+ * Get the highest value used on the axis and determine the axis width based on its length.
+ * 12 is a fairly arbitrary width here, more than needed for just the numbers to take into account commas etc.
+ */
+const getMinorAxisSize = ({
+  dataSetCategories,
+  minorAxisSize,
+  minorAxisDecimals = 0,
+  minorAxisUnit,
+}: Props) => {
```

Comment:
Add an explicit return type - this is good practice and can help prevent bugs from relying on type inference e.g. if we introduce a new conditional branch that returns a completely different type by accident, the compiler may not complain.